src/api/timer.js

```diff
@@ -0,0 +1,54 @@
+import express from 'express';
+
+export default (dispatch, action, storage) => {
+  const router = new express.Router();
+
+  router.get('/reset', async (request, response) => {
+    await dispatch(action.ResetTimer(request.timerId));
```

Comment:
Calculate if timer has been reset already, and don't issue extra reset messages.

Reply:
Problem isn't in the endpoint, but will be fixed via #46